### PR TITLE
three pages were added, scroll feature added

### PR DESCRIPTION
**Pull request by Dmytro1918**
New favicon, three pages, scroll effect

### CODE REVIEW

File: src/assets/images/favicon(1).ico
Comment (tihon-veliar) [inline src/assets/images/favicon(1).ico:1]: Please, rename this file to favicon.ico

File: src/components/FullScreenPage.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import styled from 'styled-components';
+
+
+interface FullScreenLayoutProps {
+    title: string;
+   
+  }
+
+
+const FullScreenPage: React.FC<FullScreenLayoutProps> =({title})=>{
```
Comment (tihon-veliar) [inline src/components/FullScreenPage.tsx]: Please, remove a title from the props, we aren't gonna use it. 
Also, add a "children" prop to this component, and render them in return.

File: src/components/FullScreenPage.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    return(
+        <Section>
+            <Title> {title}</Title>
+        </Section>
+    )
+}
+
+const Section=styled.div`
+width: auto;
```
Comment (tihon-veliar) [inline src/components/FullScreenPage.tsx]: Width also must be fullscreen.

**Review by tihon-veliar — COMMENTED** [review]
I wrote a couple of comments about the commit.
Have a look, please.

**Review by Dmytro1918 — COMMENTED** [review]
Fixed iaw comments. Favicon changed.

**Review by Dmytro1918 — COMMENTED** [review]
Children prop+width full screen

File: src/App.tsx [hunk 3]
```diff
@@ -1,14 +1,23 @@
...
-  const [count, setCount] = useState(0)
+const App: React.FC=() =>{
+  
 
   return (
-    <>
-      
-    </>
+    <Wrapper>
+      <FullScreenPage children/>
```
Comment (tihon-veliar) [inline src/App.tsx:11]: remove these components from here. 
We will add it when we gonna work with pages.

File: src/components/FullScreenPage.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+interface FullScreenLayoutProps {
+    children: ReactNode
+  }
+
+
+const FullScreenPage: React.FC<FullScreenLayoutProps> =({children})=>{
+    return(
+        <Section>
+            <Title> {children}</Title>
```
Comment (tihon-veliar) [inline src/components/FullScreenPage.tsx]: Delete the title component from here. 
This component(FullScreenPage) is only a wrapper for other components.

File: src/App.tsx (hunk 3 above)
Comment (tihon-veliar) [inline src/App.tsx]: In React we use "children" props not in this way. 
Also, delete those components from here, we gonna use them when we work with pages.

File: index.html [hunk 5]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="shortcut icon" type="image/x-icon" href="/src/assets/images/favicon(1).ico" />
```
Comment (tihon-veliar) [inline index.html]: You forgot to rename  a favicon(1).ico here

File: src/components/FullScreenPage.tsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        </Section>
+    )
+}
+
+const Section=styled.div`
+width: 100%;
```
Comment (tihon-veliar) [inline src/components/FullScreenPage.tsx]: we need fullscreen width -  it's better to use vw

**Review by Dmytro1918 — COMMENTED** [review]
Changes applied.

File: src/components/FullScreenPage.tsx [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+import styled from 'styled-components';
+
+
+interface FullScreenLayoutProps {
+    children: ReactNode
+  }
+
+
+const FullScreenPage: React.FC<FullScreenLayoutProps> =({})=>{
+    return(
```
Comment (tihon-veliar) [inline src/components/FullScreenPage.tsx:11]: insert "children" props here
like this

const FullScreenPage =  ({children}) => ... 
....
return 
  <Section...>
    {children}
   </Section.>
  ....

**Review by Dmytro1918 — COMMENTED** [review]
changed.

**Review by tihon-veliar — APPROVED** [review]
Approved

**Pull request merged**